lib/AddUserCmd.pm

```diff
@@ -0,0 +1,30 @@
+package AddUserCmd;
+
+use strict;
+use warnings;
+use parent 'Command';
+
+use UserService;
+use Output;
+use Error;
+
+our @ISA = qw(Command);
+
+
+sub excecute {
+
+    my $self = shift;
+    my $user_service = shift || UserService->new();
+    my ($username, $first_name, $last_name) = ($self->{'parameters'}[0],
+    $self->{'parameters'}[1], $self->{'parameters'}[2]);
+
+    my $to_return;
+    if ($user_service->add_user($username, $first_name, $last_name)) {
+        $to_return = Output->new("Usuario $username agregado" );
```

Comment:
@pauriarte told me that is only necesary to return bool, because the Handler or Command is the one that create the Output, @pauriarte is right? if not I have to change my DelUserCmd